### ISSUE TITLE
NVDA cannot upgrade old settings if winConsoleSpeakPasswords is present.

### ISSUE THREAD

**Issue by davidacm** [nvaccess/nvda#10598]
### Steps to reproduce:

1. Enable "Speak &passwords in Windows Console (may improve performance)" in advanced settings.
2. Try to install NVDA 2019.3beta1.

### Actual behavior:
An exception is generated and the current config is not updated. All current user settings are lost.

ERROR - config.ConfigManager._initBaseConf (05:37:42.736) - MainThread (12356):
Error loading base configuration
Traceback (most recent call last):
  File "config\__init__.pyc", line 424, in _initBaseConf
  File "config\__init__.pyc", line 458, in _loadConfig
  File "config\__init__.pyc", line 454, in _loadConfig
  File "config\profileUpgrader.pyc", line 24, in upgrade
  File "config\profileUpgrader.pyc", line 41, in _doConfigUpgrade
  File "config\profileUpgradeSteps.pyc", line 60, in upgradeConfigFrom_2_to_3
  File "configobj\__init__.pyc", line 502, in __getitem__
KeyError: 'terminals'
INFO - config.ConfigManager._loadConfig (05:37:42.736) - MainThread (12356):
Loading config: None

The solution is quite simple and important to solve for users when upgrade to newer NVDA's versions.
Just check if the key "terminals" exists and create it if needed in the upgradeConfigFrom_2_to_3 function.

### Expected behavior:
The current user settings are updated without losing them.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.2.1 and 2019.3beta1

#### Windows version:
Windows 10 1903

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
MSI gs65, intel i7 8750h, 32gb ram.

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Just tried with the current stable version.

**Comment by Adriani90** [comment]
closing as fixed
